src/computed/payments.js

```diff
@@ -0,0 +1,18 @@
+import { computed, extendObservable } from 'mobx';
+
+const ComputedPayments = store => {
+  extendObservable(store, {
+    computedPaymentRequest: computed(() => {
+      const { paymentsResponse } = store;
+      if (!paymentsResponse) {
+        return {};
+      }
+      return {
+        numSatoshis: paymentsResponse.num_satoshis,
+        description: paymentsResponse.description,
+      };
```

Comment:
As mentioned above, this mapping is done in the action for the other channels view. We should stay consistent. So I'm not sure we even need a computed function, since we aren't deriving new values in this case.

Reply:
I was wondering whether I needed a computed function or not. It seemed like that was the pattern you were following, so I added it, however I am new to mobx. Would I be better off just pulling from `paymentsResponse` in the view? Or should I add a new property to the store called something like `paymentRequest`?

Reply:
We used the `*Response` naming in the other actions, but `paymentRequestReponse` just reminds me of Java's `FooFactoryImplAdapter` :D But `paymentRequestReponse` is probably correct in terms of consistency's sake.